src/main/resources/application.yaml

```diff
@@ -42,3 +51,7 @@ spring:
 azure:
   application-insights:
     instrumentation-key: ${hmc.AppInsightsInstrumentationKey:00000000-0000-0000-0000-000000000000}
+jms:
+  servicebus:
+    connection-string: ${SB_CONN_STRING:Endpoint=sb://hmc-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=SendSharedAccessKey;SharedAccessKey=8EY5+1PU55K/yuWM/KEYhdVK549JBqVLXCeIcvHQFm8=;EntityPath=hmc-from-hmi-demo}
```

Comment:
Can we remove this default with real values please, should only be setting this locally as it's sensitive